### DISCUSSION TITLE
Discussion for selecting dataset for Regression Tutorial (REG103) - Level Expert

### DISCUSSION THREAD

**Discussion by Shubham-Kr-Shaw** [pycaret/pycaret#1296] [General]
I am studying and learning pycaret and was going through Regression Tutorial (REG103) - Level Expert which was under progress so If I allowed to contribute on it's progress please suggest me a data set for the  [Regression Tutorial (REG103) - Level Expert](https://github.com/pycaret/pycaret/blob/master/tutorials/Regression%20Tutorial%20Level%20Expert%20-%20REG103.ipynb
). 

Thanks & Regards 
Shubham

**Comment by ngupta23** [comment]
@moezali1 - FYI

**Comment by pycaret** [comment]
@Shubham2443 Thank you so much for looking forward to contributing. Do you have any suggestions on the dataset for the advanced tutorial?

**Reply by Shubham-Kr-Shaw** [reply]
Hello @pycaret sir, 
How about " **house.csv**  ", Which is also available in the pycaret's dataset collection. 
Or we can use a new dataset " **vehicle.csv** " which contains Used Cars data from websites, please [Click here](https://www.kaggle.com/nehalbirla/vehicle-dataset-from-cardekho?select=Car+details+v3.csv) to see the details of the dataset on Kaggle.
Or any other dataset, what is your suggestion.

Thanks & Regards
Shubham

**Reply by pycaret** [reply]
@Shubham2443 Thanks for your input. I think we might wanna choose a more sophisticated and practical dataset for Expert tutorials.
